tools/CandidateImporter.php

```diff
@@ -0,0 +1,49 @@
+<?php declare(strict_types=1);
```

Comment:
should there be a folder created in the tools directory for the importer scripts? Same as the `exporters` directory already there.

Reply:
@ridz1208

Reply:
same, we can rename it importers/exporters